Makefile

```diff
@@ -13,7 +13,13 @@ fastly:
 	@go build -trimpath $(LDFLAGS) -o "$@" ./cmd/fastly
 
 .PHONY: all
-all: fmt vet staticcheck lint gosec test build install
+all: dependencies fmt vet staticcheck lint gosec test build install
+
+.PHONY: dependencies
+dependencies:
+	go get -v -u github.com/securego/gosec/cmd/gosec
+	go get -v -u honnef.co/go/tools/cmd/staticcheck
+	go get -v -u golang.org/x/lint/golint
```

Comment:
There are some issues installing the tooling binaries this way:
- As the project is a go module, these commands will update the `go.mod` and `go.sum` manifests and therefore leaving git in a dirty state. 
- If we don't first assert whether the binaries are already on the $PATH, they have the potential to overwrite an existing binary, which some developers may not be happy about. 

Unfortunately, there isn't a easy/clean way of installing a global binary in a Go modules world. The recommended practice for this is something along the lines of:
1. cd to a directory that is (a) not a subdirectory of any directory with a go.mod file, and (b) not in GOPATH
2. go get my/tool/url
3. mv $(go env GOPATH)/bin/tool $HOME/bin
4. cd back to the original directory

In practice, step 1 is usually `cd /tmp`. One issue with above is we assume `$HOME/bin` exists and is on the $PATH. So i'm slightly inclined to think that we leave this as an exercise up to the developer instead of trying to automate via a script, as most developers of the project will already have this tooling configured locally. 

Alternatively if you think we should automate, then I suggest we create a `scripts/install-tools.sh` or similar script to do the heavily lifting I described above.

Reply:
These are all really good points. I've gone ahead and removed `dependencies` from the `all` target and updated the developer documentation with a note about how to install them.